structure_tests/command_test_v1.go

```diff
@@ -26,19 +26,20 @@ type CommandTestv1 struct {
 	EnvVars        []EnvVar
 	ExitCode       int
 	ShellMode      bool
-	Command        []string
+	Entrypoint     string
+	Args           []string
```

Comment:
I believe the Pod spec uses command / args.  Dockerfile uses Entrypoint / Cmd.

Can we make this consistent with one or the other?

Reply:
https://github.com/GoogleCloudPlatform/runtimes-common/pull/325